includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -69,12 +79,6 @@ private function getLangWithFallback( $lang ) {
 			'help' => 'http://community.wikia.com/wiki/Help:Contents',
 			'media-kit' => 'http://www.wikia.com/mediakit',
 			'media-kit-contact' => null,
-			'social-facebook' => null,
-			'social-twitter' => null,
-			'social-youtube' => null,
-			'social-instagram' => null,
-			'social-linkedin' => null,
-			'social-vk' => null,
 			'app-store' => 'https://itunes.apple.com/developer/wikia-inc./id422467077',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "app-store" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)